frontend/src/components/ProfileContent.tsx

```diff
@@ -0,0 +1,113 @@
+import * as React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { Grid, Avatar, Typography } from '@material-ui/core';
+
+// This is what the ProfileContent component expects to receive from storage.
+export type Props = {
+    username: string;
+    name: string;
+    profileImg: string;
+    gender: string;
+    religion: string;
+    location: string;
+    occupations: string[];
+    age: number | string,
+    interests: string[];
+};
```

Comment:
I think this needs a more descriptive name than `Props`. Why not `ProfileContentProps`?